jenkins-scripts/dsl/ignition.dsl

```diff
@@ -583,12 +583,7 @@ ignition_software.each { ign_sw ->
     install_default_job.with
     {
       // disable some bottles
-      if (("${ign_sw}" == "cmake" && "${major_version}" == "1") ||
-          ("${ign_sw}" == "common" && "${major_version}" == "2") ||
-          ("${ign_sw}" == "fuel-tools" && "${major_version}" == "2") ||
-          ("${ign_sw}" == "gui" && "${major_version}" == "0") ||
-          ("${ign_sw}" == "math" && "${major_version}" == "5") ||
-          ("${ign_sw}" == "msgs" && "${major_version}" == "2") ||
+      if (("${ign_sw}" == "gui" && "${major_version}" == "0") ||
           ("${ign_sw}" == "transport" && "${major_version}" == "5"))
```

Comment:
Can't transport5 be removed too?

Reply:
yeah, I meant to remove that too

Reply:
4f384fe7d8cf7523c195b1c4ba2b8c0e0109c858

[![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=_dsl_ignition&build=1157)](https://build.osrfoundation.org/job/_dsl_ignition/1157/) https://build.osrfoundation.org/job/_dsl_ignition/1157/